packages/features/ee/workflows/lib/alphanumericSenderIdSupport.ts

```diff
@@ -84,4 +84,5 @@ const noAlphanumericSenderIdSupport = [
   "+58",
   "+84",
   "+260",
+  "+61",
```

Comment:
Isn't there an API from Twilio that we can use and cache the response for?
It might be overkill if it's a rare change though. Also, how does it work if someone uses that alphanumeric ID and then it longer stops being supported. What happens to all such workflows.

Reply:
It should then just fall back to our normal twilio number